app/components/shared/searchable_collection_component/searchable_collection_component.scss

```diff
@@ -3,27 +3,27 @@
     margin-bottom: 0;
   }
 
-  &.scrollable {
+  &.collection-component--scrollable {
     .govuk-form-group {
-      max-height: 194px;
+      max-height: 400px;
```

Comment:
this makes the height of the subjects select larger but have to live with it for the moment, i dont see it as a huge problem. chris more keen on at least 5 schools being visible